virttest/libvirt_cgroup.py

```diff
@@ -470,7 +470,7 @@ def set_cpuset_cpus(self, value):
         cmd = "echo %s > /sys/fs/cgroup/cpuset/machine.slice/cpuset.cpus" % value
         process.run(cmd, ignore_status=False, shell=True)
 
-    def get_cpuset_cpus(self):
+    def get_cpuset_cpus(self, vm_name):
```

Comment:
I think set_cpuset_cpus also need a check for "/sys/fs/cgroup/cpuset/machine.slice/cpuset.cpus"
others LGTM. Thanks.

Reply:
updated